src/figuresClasses.ts

```diff
@@ -1,11 +1,81 @@
-export interface Figure {}
+type Shape = `triangle` | `circle` | `rectangle`;
```

Comment:
```suggestion
type Shape = 'triangle' | 'circle' | 'rectangle';
```
use regular strings here